shared/utils.js

```diff
@@ -3,7 +3,12 @@
 // TODO: make all of these functions configurable from the outside
 // the user should be able to manipulate those
 
-function getNamespace(service, stage) {
+function getNamespace(serverless, stage) {
+  const { namespace } = serverless.service.provider.k8s
```

Comment:
`k8s` object will be there only if user has it configured in it's yaml. Therefore above will crash for services with this not configured

Reply:
fixed!